templates/home.html

```diff
@@ -3,7 +3,7 @@
 {% block title %}Home{% endblock %}
 {% block body %}
 <h4>
-    This is the website for <strong>{{ bot.name }}</strong> that moderates <a href="http://twitch.tv/{{ streamer.full_name }}">{{ streamer.name }}'s</a> channel.
+    This is the website for <strong>{{ bot.name }}</strong> that moderates <a href="http://twitch.tv/{{ streamer.full_name }}">{{ streamer.name }}'{{% "" if streamer.name.endswith('s') else "s" %}}</a> channel.
```

Comment:
```suggestion
    This is the website for <strong>{{ bot.name }}</strong> that moderates <a href="http://twitch.tv/{{ streamer.full_name }}">{{ streamer.name }}'{{% "" if lower(streamer.name).endswith('s') else "s" %}}</a> channel.
```

Reply:
Wouldn't it be best to use `streamer.full_name`, since that pulls it from the main part of the config as opposed to `streamer.name` that uses the streamer's display name?

Reply:
Actually, we have to use `streamer.full_name`, else we could be using asian display names and error out as a result.